lib/Runtime/Library/ConcatString.cpp

```diff
@@ -200,6 +200,14 @@ namespace Js
         this->SetBuffer(propertyRecord->GetBuffer());
     }
 
+    const void * LiteralStringWithPropertyStringPtr::GetOriginalStringReference()
```

Comment:
Is this the right type? I thought that `const void *` is a different type to `void const *`? Or is that only to `void * const`...

Reply:
They're the same, but I'll go make them visually consistent. Thanks! (The only semantically different option would be `void * const`.)